microcosm/opaque.py

```diff
@@ -39,9 +46,52 @@ def __enter__(self):
             self.saved = deepcopy(opaque._store)
             opaque.update(self.func())
 
+            # Use of a tracing solution on top of opaque is optional!
+            if opaque.tracer:
```

Comment:
When is this injected into opaque?

Reply:
when the factory is loaded. So basically when anything imports microcosm because I added the tracer as a factory.